x-pack/plugins/cloud_security_posture/public/pages/rules/index.tsx

```diff
@@ -52,43 +52,35 @@ const getRulesBreadcrumbs = (
   return breadCrumbs;
 };
 
-const isPolicyTemplate = (name: unknown): name is keyof CloudPostureIntegrations =>
-  typeof name === 'string' && name in cloudPostureIntegrations;
-
 const getRulesSharedValues = (
   packageInfo?: PackagePolicy
 ): NonNullable<EuiDescriptionListProps['listItems']> => {
-  const enabledInput = packageInfo?.inputs.find((input) => input.enabled);
-  if (!enabledInput || !isPolicyTemplate(enabledInput.policy_template)) return [];
-
-  const integration = cloudPostureIntegrations[enabledInput.policy_template];
-  const enabledIntegrationOption = integration.options.find(
-    (option) => option.type === enabledInput.type
-  );
+  const enabledIntegration = getEnabledCspIntegrationDetails(packageInfo);
+  const values = [];
```

Comment:
```suggestion
  const values: Array<{title: string, description: string> = [];
```
or better yet, the actual type

i'm not sure what the return type of `getRulesSharedValues` without it, does it pick it up or is it `never[]` ?